pkg/api/nsmdapi/gen.go

```diff
@@ -0,0 +1,22 @@
+// Copyright 2018 Red Hat, Inc.
```

Comment:
Please do not pull this API in... putting nsmdp in a separate container and talking over GRPC was unecessarily complex, and we can simply make the NSMgr provide the K8s DP API itself.